readme.md

```diff
@@ -57,6 +57,7 @@ The `nodeSelector` selects the nodes observed by this operator, which are also t
 The `watermarkPercentage` define the percentage of capacity of user workload that will be allocated to low priority pods. So in this example 20% of the user allocated capacity will be allocated via low priority pods. This also means that when the user workload reaches 80% capacity of the nodes selected by this CR (and the autoscaler), the cluster will start to scale.
 
 ## Deploying the Operator
+> :lock: **Disconnected Installation Supported**
```

Comment:
this is true only for OLM installation so I' like to move this to that section. Also can we remove the lock, it does not feel like the right icon anyway. Can we just say
**Note**: This operator supports being installed in Disconnected Installations

Reply:
@raffaelespazzoli updated